kobo/django/xmlrpc/models.py

```diff
@@ -7,7 +7,8 @@
 
 class XmlRpcLog(models.Model):
     dt_inserted = models.DateTimeField(auto_now_add=True)
-    user        = models.ForeignKey(settings.AUTH_USER_MODEL, null=True, blank=True)
+    user        = models.ForeignKey(settings.AUTH_USER_MODEL, null=True, blank=True,
+                                    on_delete=models.SET_NULL)
```

Comment:
This is a functional change because the default is CASCADE.  Is this intentional?

It seems like the kind of thing that should be mentioned in the commit message.

Reply:
Setting `XmlRpcLog.user` to NULL would be flexible, at least, the logs are still there and can be checked for some purposes.